partcad/src/partcad/context.py

```
@@ -716,60 +615,63 @@ async def find_suppliers(self, cart: ProviderCart) -> dict[str, list[str]]:
+            suppliers[name] = suppliers_per_part
 
-            if f"{project_name}:{part_name}" not in suppliers:
-                pc_logging.error(
-                    f"No supplier found for {project_name}:{part_name}"
-                )
+        # TODO(clairbee): calculate the recommended suppliers and reorder the results accordingly
+        return suppliers
 
-        # TODO(clairbee): calculate the recommended suppliers and
-        #                 reorder the results accordingly
+    async def find_part_suppliers(self, part_item: ProviderCartItem, cart: ProviderCart = None) -> dict[str, list[str]]:
```

Ruff (0.8.2)

`ProviderCartItem` may be undefined, or defined from star imports

(F405)